recipes-core/aziotd/aziotd_%.bbappend

```diff
@@ -0,0 +1,11 @@
+target = "${@d.getVar('TARGET_SYS',True).replace('-', ' ')}"
+BINDGEN_EXTRA_CLANG_ARGS = "${@bb.utils.contains('target', 'arm', \
+                              '--sysroot=${WORKDIR}/recipe-sysroot -I${WORKDIR}/recipe-sysroot/usr/include -mfloat-abi=hard', \
+                              '--sysroot=${WORKDIR}/recipe-sysroot -I${WORKDIR}/recipe-sysroot/usr/include', \
+                              d)}"
+export BINDGEN_EXTRA_CLANG_ARGS
+
+# Copy keys.generated.rs
+do_compile_prepend () {
```

Comment:
Is this still needed? Thought it got handled by adding aziot-keys ...

Reply:
Yes this is needed. This file gets used first for aziot-keys